client/my-sites/pages/page.jsx

```diff
@@ -195,14 +195,14 @@ module.exports = React.createClass( {
 			return (
 				<PopoverMenuItem className="page__trash-item" onClick={ this.updateStatusTrash }>
```

Comment:
@omarjackman Nope, they need to be maintained as separate. The `page__trash-item` doesn't actually add any CSS, and the `page__delete-item` needs to be maintained as stylistically-distinct from `page__trash-item`. Basically, the `page__trash-item` is an unnecessary className for CSS.